wdl-parsing/WDLParsingFunction/pom.xml

```diff
@@ -0,0 +1,305 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>io.dockstore</groupId>
+    <artifactId>wdl-parser</artifactId>
+    <version>1.0</version>
+    <packaging>jar</packaging>
+    <name>WDL parser created for SAM CLI.</name>
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
Consider targeting Java 11
Java 8 seems to be slower when invoking/starting as a lambda
https://www.nuweba.com/blog/benchmarking-invocation-overhead-on-new-aws-lambda-runtimes